streampark-common/src/main/java/org/apache/streampark/common/domain/ResponseCode.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.streampark.console.base.domain;
+package org.apache.streampark.common.domain;
```

Comment:
hi, 

Thanks for your contribution! It's a great improvement. However, from my perspective, there are a issue:

The common layer is a globally module where we strive to include common functionalities that are unrelated to specific business logic. Besides providing a data platform based on Spring Boot, StreamPark is also a stream processing development framework. We have provided some Flink|spark rapid development APIs, allowing users to independently leverage the capabilities of the development framework. Looking at it this way, it is not reasonable to place platform-specific logic such as HTTP status codes and backend exception message display in the common layer. These are not common capabilities, and they are not needed by the development framework.